### ISSUE TITLE
Login error: Invalid response after nss update to version 3.58

### ISSUE THREAD

**Issue by necropresto** [EionRobb/purple-hangouts#225]
Hi! I'm receiving "hangouts - Login error: Invalid response" after update nss to version 3.58.

Follow some debug lines from connection:

DEBUG proxy: Attempting connection to 2800:3f0:4004:809::200a
DEBUG proxy: Connecting to www.googleapis.com:443 with no proxy
DEBUG proxy: Connection in progress
DEBUG proxy: Connecting to www.googleapis.com:443.
DEBUG proxy: Connected to www.googleapis.com:443.
DEBUG nss: Handshake failed  (-12251)
DEBUG socket: invalid state: 3 (should be: 2)DEBUG http: Request 0x557f39815100 performed without success.

**Comment by lekzz** [comment]
Seems to be a libnss3 problem as the discord plugin also stopped working. Thanks for the headups that it was libnss3.

**Comment by EionRobb** [comment]
Yep, it's an issue with NSS. https://bugzilla.mozilla.org/show_bug.cgi?id=1672703

Downgrading to the previous version should resolve it until they put out their next point release that fixes it

**Comment by alevinetx** [comment]
sooo... i'm going to ask the obtuse question:  as a hangouts user who got the connection problems today and upgraded to the newest pidgin version to try and resolve it.. and i'm seeing the "3 s/b 2" message..  What should I be doing to ```downgrade to the previous version```  ?

**Comment by Thaodan** [comment]
Whats the latest version to you?

**Comment by alevinetx** [comment]
for me?

NSS Preferences 2.14.1

Pidgin 2.14.1 (libpurple 2.14.1)
f902de8a1b5eac32c33c63c95b40693ba5c4aed8

**Comment by Thaodan** [comment]
I have the same version on Arch, check your NSS if it contains the mentioned 
fix.

**Comment by alevinetx** [comment]
that brings me back to the comment about how one should downgrade.   downgrade which part, how

**Comment by EionRobb** [comment]
Another workaround is to use the "NSS Preferences Plugin" in Tools->Plugins to set the max tls version to TLS 1.2

**Comment by BoFiS** [comment]
Hmm, I have NSS Preferences Plugin and tried setting Max TLS 1.2 but that has not resolved the issue this time around, also what should the min version be set to?
